lib/board.rb

```diff
@@ -115,7 +115,18 @@ def overlapping?(ship, coordinate)
         cell.ship != ship
     end
 
-    # def overlapping?
-    # end
-    
+    def render(show_ship = false)
+        board_display = "  1 2 3 4 \n"
+        ('A'..'D').each do |letter|
+            row_display = letter + ' '
+            (1..4).each do |number|
+                coordinate = "#{letter}#{number}"
+                row_display += cells[coordinate].render(show_ship) + ' '
+            end
+            board_display += row_display.rstrip + " \n"
```

Comment:
I don't think we need rstrip here?

Reply:
Your right! Lets refactor that. We can just remove the space in front of the " /n" and get rid of rstrip